src/server/acl/acl_family.cc

```diff
@@ -468,6 +469,37 @@ void AclFamily::GetUser(CmdArgList args, ConnectionContext* cntx) {
   (*cntx)->SendSimpleString(acl);
 }
 
+void AclFamily::GenPass(CmdArgList args, ConnectionContext* cntx) {
+  if (args.length() > 1) {
+    (*cntx)->SendError(facade::UnknownSubCmd("GENPASS", "ACL"));
+    return;
+  }
+  uint32_t random_bits = 256;
+  if (args.length() == 1) {
+    auto requested_bits = facade::ArgS(args, 0);
+
+    if (!absl::SimpleAtoi(requested_bits, &random_bits) || random_bits == 0 || random_bits > 4096) {
+      return (*cntx)->SendError(
+          "ACL GENPASS argument must be the number of bits for the output password, a positive "
+          "number up to 4096");
+    }
+  }
+  std::random_device urandom("/dev/urandom");
+  const size_t result_length = (random_bits + 3) / 4;
+  constexpr size_t step_size = sizeof(decltype(std::random_device::max()));
```

Comment:
@kostasrim hey, I changed this a bit, since`max` is a static member. Is this correct?